src/main/java/ReadFile.java

```diff
@@ -0,0 +1,59 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ReadFile {
+    /** Destination path of the duke.txt storage file. */
+    private String path;
+
+    /**
+     * Construct new ReadFile object.
+     *
+     * @param filePath Destination path of the duke.txt storage file.
+     */
+    public ReadFile(String filePath) {
+        path = filePath;
+    }
+
+    /**
+     * Returns the number of lines in the duke.txt storage file.
+     *
+     * @return Number of lines.
+     * @throws IOException If input or output exceptions are encountered.
+     */
+    private int readLines() throws IOException {
+        FileReader reader = new FileReader(path);
+        BufferedReader bufferedReader = new BufferedReader(reader);
+
+        String nextLine;
+        int numLines = 0;
+
+        while ((nextLine = bufferedReader.readLine()) != null) {
+            ++numLines;
+        }
+
+        bufferedReader.close();
```

Comment:
Might be better to wrap this into either a try-with-resources or try-catch-finally, so that the `.close()` will be run regardless of whether an exception occurs or not. In this case, when a IOException occurs in the previous lines, the `bufferedReader.close()` won't actually be executed.